Makefile

```diff
@@ -1,3 +1,8 @@
 update_metadata:
 	(cd libPhoneNumberTests && ./metadataGenerator) && \
-		(cd libPhoneNumber && ./GeneratePhoneNumberHeader.sh)
+		(cd libPhoneNumber && ./GeneratePhoneNumberHeader.sh) && \
+		(cp libPhoneNumber/NBGeneratedPhoneNumberMetaData.h MetaDataPlistCreator) && \
```

Comment:
Since this is running locally and one-off now, do we need the header at all? Maybe we can have the tool read from the JSON directly.

Reply:
I didn't want to make the changes bigger than necessary. Creating the .h file is now an intermediate step (triggered and executed by "well tested" code). We can refactor and rewrite all of this stuff, but is it worth it?

Reply:
I don't think it would make the changes any bigger. All of the JSON loading is wrapped up in one function already; that function gets replaced with a call to NSJSONSerialization.

Reply:
The expensive deduplication is written in objective-c. Is there a cheap and safe way to include this in a swift file which is executed from the shell? If not, we have to rewrite the existing code. That's what I tried to say.